OOModel/src/declarations/Class.cpp

```diff
@@ -228,4 +239,22 @@ Class* Class::expressionToClass(Expression* expr)
 	return nullptr;
 }
 
+QSet<Class*> Class::directSubClasses()
+{
+	auto top = root();
+	//Find all the classes where this method is subclasses
+	QSet<Class*> result;
+	QList<Model::Node*> toCheck;
+	toCheck.append(top);
+	while (!toCheck.isEmpty())
+	{
+		auto check = toCheck.takeLast();
+		if (auto clazz = DCast<Class>(check))
```

Comment:
Don't use hacks like `clazz`. Use proper English. Here you can say: `someClass`.